src/documentation/markdown.rs

```diff
@@ -61,6 +62,14 @@ impl<'a> PlainOverlay<'a> {
     fn valid_url(url: &str) -> bool {
         url::Url::parse(url).is_ok()
     }
+    fn reachable_link(url: &str) -> bool {
+        use reqwest;
+        let response = reqwest::blocking::get(url).unwrap();
```

Comment:
If we want this to remain blocking, which I think makes sense, we should think about parallelizing work per document eventuall (separate PR).

Reply:
We should also track which url we already checked, we don't want to spam a server with requests and on the other side cause the user to wait for an extended time for not much benefit.